File: src/api/EProfspilka.Core/Entities/Event.cs
using EProfspilka.Core.Entities.Base;
using EProfspilka.Core.Enumerations;

namespace EProfspilka.Core.Entities;

public class Event : BaseEntity
{
	public string Title { get; set; }

	public DateTime Date { get; set; }

	public Status Status { get; set; }

	public ICollection<EventImage> EventImages { get; set; }

	public string Description { get; set; }

	public string ShortDescription { get; set; }
}

File: src/web/src/models/event.ts
export enum EventStatus {
    Draft = 'draft',
    Published = 'published',
}

export interface Event {
    id: string;
    title: string;
    description: string;
    date: string | null;
    isPassed: boolean;
    shortDescription: string;
    images: string[];
    status: EventStatus;
}